src/Config/CS_SDK.props

```diff
@@ -59,6 +59,10 @@
       <PrivateAssets>all</PrivateAssets>
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
+    <ReferenceCopyLocalPaths>
```

Comment:
what local directory does this refer to?

Reply:
A new folder that will contain all the package/assembly references of test projects
`D:\dev\dynamo\DynamoDS\test\test_dependencies`
The purpose of this is to avoid polluting the bin folder with test dependencies

Reply:
It is based on the parent directory
```
<TestProjectDefaults Condition="$(MSBuildProjectDirectory.Contains('test'))">true</TestProjectDefaults>
<ItemDefinitionGroup Condition=" '$(TestProjectDefaults)' == 'true' ">
```